packages/v4/patternfly-docs/content/design-guidelines/components/data-list/data-list.md

```diff
@@ -31,20 +45,7 @@ See [when to use compact vs. default spacing](#compact-vs-default-spacing) for m
 
```

Comment:
The "when to use compact vs. default spacing" links needs to change, because it doesn't match the header right now
Try: (#when-to-use-a-data-list-vs-a-table) instead